internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -149,7 +151,7 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				placeholderReplacer := strings.NewReplacer("{account_id}", accountID, "{zone_id}", zoneID)
 				endpoint = placeholderReplacer.Replace(endpoint)
 
-				client := cloudflare.NewClient()
```

Comment:
do we need to rename it here? we can also just alias `cloudflare-go` to something else instead of reassigning here.